README.md

```diff
@@ -46,18 +46,19 @@ var_dump($meta);
 
 ## Parsers
 
-There are 2 parsers included in package and you can crate new implementing interface `Tomaj\Scraper\Parser\ParserInterface`.
+There are 3 parsers included in package and you can create new implementing interface `Tomaj\Scraper\Parser\ParserInterface`.
 
-2 parsers:
- - `Tomaj\Scraper\Parser\OgParsers` - based on og meta attributes in html
+3 parsers:
+ - `Tomaj\Scraper\Parser\OgParsers` - based on og (Open Graph) meta attributes in html (built on regular expressions)
+ - `Tomaj\Scraper\Parser\OgDomParsers` - also based on og (Open Graph) meta attributes in html (built on php DOM extension)
```

Comment:
what about removing last `s` (I know it is in OgParsers but it was mistake). I will change it in next major release

Reply:
sure